src/cgroup.rs

```diff
@@ -492,6 +492,34 @@ impl Cgroup {
         v.dedup();
         v
     }
+
+    /// Checks if the cgroup exists.
+    ///
+    /// Returns true if at least one subsystem exists.
+    pub fn exists(&self) -> bool {
+        for subsystem in self.subsystems().iter() {
+            let exists = match subsystem {
+                Subsystem::Pid(c) => c.exists(),
+                Subsystem::Mem(c) => c.exists(),
+                Subsystem::CpuSet(c) => c.exists(),
+                Subsystem::CpuAcct(c) => c.exists(),
+                Subsystem::Cpu(c) => c.exists(),
+                Subsystem::Devices(c) => c.exists(),
+                Subsystem::Freezer(c) => c.exists(),
+                Subsystem::NetCls(c) => c.exists(),
+                Subsystem::BlkIo(c) => c.exists(),
+                Subsystem::PerfEvent(c) => c.exists(),
+                Subsystem::NetPrio(c) => c.exists(),
+                Subsystem::HugeTlb(c) => c.exists(),
+                Subsystem::Rdma(c) => c.exists(),
+                Subsystem::Systemd(c) => c.exists(),
```

Comment:
should we add a default arm check?

Reply:
All supported subsystems are listed in this match. IMO, a default arm check is unnecessary.